lisp_ease/src/queries/items.lisp

```diff
@@ -0,0 +1,71 @@
+; Defines all the items (objects) which have been used within the Virtual Reality and the axis object,
+; which are now added here, so that the same items can be used in the bullet world.
+; All of these objetcs and their type are prefixed with 'ba-*' so that it can be differentiated between
+; these objects and the objects otherwise present in the bullet world environment.
+; The objects are NOT spawned at the 0 0 0 coordinate. They each have different spawning points.
+(in-package :le)
+
+; List of all the .stl files, of the newly added items. aka list of meshes. 
+(defparameter *mesh-files*
+  '((:ba-bowl "package://lisp_ease/resource/ba_bowl.stl" t)
+    (:ba-cup "package://lisp_ease/resource/ba_cup.stl" t)
+    (:ba-muesli "package://lisp_ease/resource/ba_muesli.stl" t)
+    (:ba-fork "package://lisp_ease/resource/ba_fork.stl" t)
+    (:ba-milk "package://lisp_ease/resource/ba_milk.stl" t)
+    (:ba-axes "package://lisp_ease/resource/ba_axes.stl" t)))
+
+; appends the newly defined objects in the *mesh-files* variable to the *mesh-files*
+; of btr, so that they can be loaded
+(defun append-meshes-to-list ()
+  "Appends the newly defined objects in the *mesh-files* variable to the *mesh-files* of btr, so that they can be loaded in the bullet world."
+     (btr:add-objects-to-mesh-list "lisp_ease"))
```

Comment:
If this does not work or compile, add the second argument -- directory where to search for cad models. In your case -- "resource".